source/develop/release-management/features/XenToOvirt.html.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Importing Xen on REHL 5.x to oVirt
+category: feature
+authors: Shahar Havivi
+wiki_category: Feature|v2v
+wiki_title: Features/XenToOvirt
+wiki_revision_count: 1
+wiki_last_updated: 2015-06-02
+feature_name: 'v2v: Importing Xen on RHEL to oVirt'
+feature_modules: all
+---
+
+# Importing Xen on RHEL 5.x VMs to oVirt
+-----------------------------------------------------------
+### Summary
+oVirt had the ability to import VMs from other hypervisor including **Xen** on RHEL 5.x (not yet for Citrix Xen)
+The Import process uses [virt-v2v][1] (under the "INPUT FROM RHEL 5 XEN" section) which explain the prerequisites that are needed in order to import Xen VMs.
+[1]: http://libguestfs.org/virt-v2v.1.html
+
+### Importing VM
+In order to import VMs  password-less SSH access has to be enabled between VDSM host and the Xen host.
+The following steps must be taken under vdsm user in the VDSM host.
+- You need to enable vdsm user, log in as root and run the following:
+ ```
+
+$ mkdir /home/vdsm
+$ chown 36:36 /home/vdsm
```

Comment:
Why is creating the home directory in /home needed? Has anyone specifically asked for this?

Because vdsm user already has home set to /var/lib/vdsm. Can't we break anything by changing it?